zerver/webhooks/helloworld/doc.md

```diff
@@ -23,13 +23,13 @@ integration](/api/incoming-webhooks-walkthrough).
         (zulip-server) vagrant@vagrant:/srv/zulip$
         ./manage.py send_webhook_fixture_message \
         > --fixture=zerver/tests/fixtures/helloworld/hello.json \
-        > '--url=http://localhost:9991/api/v1/external/helloworld?api_key=abcdefgh&stream=channel%20name;'
+        > '--url={{ api_url }}{{ integration_url }}?api_key=abcdefgh&stream=channel%20name;'
```

Comment:
I think this change is probably wrong -- these docs are for writing a new integration, and that command expects to be used with the development environment server.